cmd/podman/containers/run.go

```diff
@@ -63,7 +64,7 @@ func runFlags(cmd *cobra.Command) {
 
 	flags.SetNormalizeFunc(utils.AliasFlags)
 	flags.BoolVar(&runOpts.SigProxy, "sig-proxy", true, "Proxy received signals to the process")
-	flags.BoolVar(&runRmi, "rmi", false, "Remove container image unless used by other containers")
+	flags.BoolVar(&runRmi, "rmi", false, "Remove image unless used by other containers, implies --rm=true")
```

Comment:
I think just `implies --rm` is short and easier to understand for most users. I don't think most people know about the extra `=true/false` syntax.